src/modules/home/index.tsx

```diff
@@ -4,6 +4,7 @@ import { SinglePropertyRoute } from './routes/single-property.route';
 import { AuthLayout } from '#/routes/layout-protected.route';
 import { CreateAccommodationRoute } from '#/modules/home/routes/create-acccommodation.route.tsx';
 import { PaymentResult } from './components/payment';
+import { BookingsPageRoute } from './routes/bookings-page.route';
```

Comment:
use lazyRoute. we already have large bundle size, your current approach will increase it more